### ISSUE TITLE
Bug in test_empty_files and test_empty_files_sequence_length?

### ISSUE THREAD

**Issue by szhan** [tskit-dev/tskit#2373]
Are `test_empty_files` and `test_empty_files_sequence_length` in `test_highlevel.py` supposed to cover instances when an individuals file is empty? If so, then it may be a bug.

Right now, the code is:
```
def test_empty_files(self):
    nodes_file = io.StringIO("is_sample\ttime\n")
    edges_file = io.StringIO("left\tright\tparent\tchild\n")
    sites_file = io.StringIO("position\tancestral_state\n")
    mutations_file = io.StringIO("site\tnode\tderived_state\n")
    migrations_file = io.StringIO("left\tright\tnode\tsource\tdest\ttime\n")
    with pytest.raises(_tskit.LibraryError):
        tskit.load_text(
            nodes=nodes_file,
            edges=edges_file,
            sites=sites_file,
            mutations=mutations_file,
            migrations=migrations_file,
        )

def test_empty_files_sequence_length(self):
    nodes_file = io.StringIO("is_sample\ttime\n")
    edges_file = io.StringIO("left\tright\tparent\tchild\n")
    sites_file = io.StringIO("position\tancestral_state\n")
    mutations_file = io.StringIO("site\tnode\tderived_state\n")
    migrations_file = io.StringIO("left\tright\tnode\tsource\tdest\ttime\n")
    ts = tskit.load_text(
                nodes=nodes_file,
                edges=edges_file,
                sites=sites_file,
                mutations=mutations_file,
                migrations=migrations_file,
                sequence_length=100,
    )
    assert ts.sequence_length == 100
    assert ts.num_nodes == 0
    assert ts.num_edges == 0
    assert ts.num_sites == 0
    assert ts.num_edges == 0
    assert ts.num_migrations == 0
```

I think that it should be:
```
def test_empty_files(self):
    individuals_files = io.StringIO("flags\n") # FIX
    nodes_file = io.StringIO("is_sample\ttime\n")
    edges_file = io.StringIO("left\tright\tparent\tchild\n")
    sites_file = io.StringIO("position\tancestral_state\n")
    mutations_file = io.StringIO("site\tnode\tderived_state\n")
    migrations_file = io.StringIO("left\tright\tnode\tsource\tdest\ttime\n")
    with pytest.raises(_tskit.LibraryError):
        tskit.load_text(
            individuals=individuals_file, # FIX
            nodes=nodes_file,
            edges=edges_file,
            sites=sites_file,
            mutations=mutations_file,
            migrations=migrations_file,
        )

def test_empty_files_sequence_length(self):
    individuals_files = io.StringIO("flags\n") # FIX
    nodes_file = io.StringIO("is_sample\ttime\n")
    edges_file = io.StringIO("left\tright\tparent\tchild\n")
    sites_file = io.StringIO("position\tancestral_state\n")
    mutations_file = io.StringIO("site\tnode\tderived_state\n")
    migrations_file = io.StringIO("left\tright\tnode\tsource\tdest\ttime\n")
    ts = tskit.load_text(
                individuals=individuals_file, # FIX
                nodes=nodes_file,
                edges=edges_file,
                sites=sites_file,
                mutations=mutations_file,
                migrations=migrations_file,
                sequence_length=100,
    )
    assert ts.sequence_length == 100
    assert ts.num_individuals == 0 # FIX
    assert ts.num_nodes == 0
    assert ts.num_edges == 0
    assert ts.num_sites == 0
    assert ts.num_mutations == 0
    assert ts.num_migrations == 0
```

**Comment by szhan** [comment]
I'll fix this once issue #2366 is finished.

**Comment by szhan** [comment]
I also added

`assert ts.num_mutations == 0`

in `test_empty_files_sequence_length`.
